common/protocols/src/main/proto/app/coronawarn/server/common/protocols/internal/v2/app_config_android.proto

```diff
@@ -25,6 +26,8 @@ message ApplicationConfigurationAndroid {
 
   DiagnosisKeysDataMapping diagnosisKeysDataMapping = 8;
   DailySummariesConfig dailySummariesConfig = 9;
+
+  PPDDEventDrivenUserSurveyParametersAndroid eventDrivenUserSurveyParameters = 10;
```

Comment:
@mlenkeit but the app-config changes are relevant for this repo, aren't they!?

Reply:
good point!